README.md

```diff
@@ -12,6 +12,7 @@ $ bower install gaia-components/gaia-component
 
 ```js
 var MyComponent = component.register('my-component', {
+  extends: HTMLButtonElement.prototype,
```

Comment:
extends from native element will inherit native element behavior and improve the accessibility